rome/src/main/java/com/rometools/rome/io/impl/RSS091UserlandParser.java

```diff
@@ -148,6 +143,8 @@ protected WireFeed parseChannel(final Element rssRoot, final Locale locale) {
             channel.setSkipDays(skipDays);
         }
 
+        channel.setItems(parseItems(rssRoot, locale));
```

Comment:
Same as my previous comment - should work without it?

Reply:
Yes, removed.